vcr/verifier/interface.go

```diff
@@ -48,8 +48,8 @@ type Verifier interface {
 	RegisterRevocation(revocation credential.Revocation) error
 
 	// VerifyVP verifies the given Verifiable Presentation. If successful, it returns the credentials within the presentation.
-	// If verifyVCs is true, it will also verify the credentials inside the VP, checking their correctness, signature and trust status.
-	VerifyVP(presentation vc.VerifiablePresentation, verifyVCs bool, validAt *time.Time) ([]vc.VerifiableCredential, error)
+	// If verifyVCs is true, it will also verify the credentials inside the VP, checking their correctness, signature and trust status (if allowUntrustedVCs is true).
+	VerifyVP(presentation vc.VerifiablePresentation, verifyVCs bool, allowUntrustedVCs bool, validAt *time.Time) ([]vc.VerifiableCredential, error)
```

Comment:
Probably better to make this a list of types, because if the presentation contained both the org and employee credential, the org should be verified.

Reply:
I think the potential use cases outclass any reasonable changes we can make to this signature without going wildly out of scope. E.g.; maybe not all VCs need to be verified?

The presentation type (and validator) specifies there's exactly 1 VC, which is the NutsEmployeeCredential. Redesign would probably lead to a flexible DSL/business rule engine to express these.

Reply:
Yeah, perhaps? I originally designed this credential to be presented together with the orgCredential in one presentation. Somewhere this design got lost in translation/adaptation. Now we have to find the orgCredential from the store with custom logic. So yes, it is already weird indeed. 

> why call Verify on a presentation and not verify the VCs itself?
Verification of the VP is necessary because the credentials inside it must have the samen subject as the presentation issuer.